src/main/java/adris/altoclef/TaskCatalogue.java

```diff
@@ -218,7 +218,10 @@
             shapedRecipe3x3Block("emerald_block", Items.EMERALD_BLOCK, "emerald");
             shapedRecipe3x3Block("lapis_block", Items.LAPIS_BLOCK, "lapis_lazuli");
             shapedRecipe3x3Block("slime_block", Items.SLIME_BLOCK, "slime_ball");
+            shapedRecipe3x3Block("bone_block", Items.BONE_BLOCK, "bone_meal");
+            shapedRecipe3x3Block("melon", Items.MELON, "melon_slice").dontMineIfPresent();
             shapedRecipe2x2Block("glowstone", Items.GLOWSTONE, "glowstone_dust").dontMineIfPresent();
+            shapedRecipe2x2Block("clay", Items.CLAY, "clay_ball").dontMineIfPresent();
```

Comment:
Works as expected. I did get this error on testing though:
![image](https://user-images.githubusercontent.com/72313560/148275016-78676665-9b28-4003-a220-535544cde073.png)

However, I believe this error is caused by interacting with crafting tables that are not properly accounted for. I do not believe it would show up in deployment.

Further tests without a table executed properly without issue or error.

![image](https://user-images.githubusercontent.com/72313560/148275153-5a6b9cd3-a5ac-4d25-830c-4a85226c301e.png)